src/main/java/model/lotto/LottoNumbers.java

```diff
@@ -0,0 +1,19 @@
+package model.lotto;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class LottoNumbers {
+    private static final List<Integer> lottoNumbers = new ArrayList<>();
+    private static final int LOTTO_MIN_NUMBER = 1;
+    private static final int LOTTO_MAX_NUMBER = 45;
+
+    static {
+        IntStream.range(LOTTO_MIN_NUMBER, LOTTO_MAX_NUMBER).forEach(lottoNumbers::add);
```

Comment:
한줄에 점 하나씩으로 작성하면 가독성이 더 높아집니다 :) 
```suggestion
        IntStream.range(LOTTO_MIN_NUMBER, LOTTO_MAX_NUMBER)
        .forEach(lottoNumbers::add);
```